labelbox/schema/annotation_import.py

```diff
@@ -0,0 +1,294 @@
+
+        Returns:
+            List of dicts containing error messages. Empty list means there were no errors
+            See `AnnotationImport.statuses` for more details.
+
+        * This information will expire after 24 hours.
+        """
+        self.wait_until_done()
+        return self._fetch_remote_ndjson(self.error_file_url)
+
+    @property
+    def statuses(self) -> List[Dict[str, Any]]:
+        """
+        Status for each individual annotation uploaded.
+
+        Returns:
+            A status for each annotation if the upload is done running.
+            See below table for more details
+
+        .. list-table::
+           :widths: 15 150
+           :header-rows: 1
+
+           * - Field
+             - Description
+           * - uuid
+             - Specifies the annotation for the status row.
+           * - dataRow
+             - JSON object containing the Labelbox data row ID for the annotation.
+           * - status
+             - Indicates SUCCESS or FAILURE.
+           * - errors
+             - An array of error messages included when status is FAILURE. Each error has a name, message and optional (key might not exist) additional_info.
+
+        * This information will expire after 24 hours.
+        """
+        self.wait_until_done()
+        return self._fetch_remote_ndjson(self.status_file_url)
+
+    def wait_until_done(self, sleep_time_seconds: int = 5) -> None:
+        """Blocks import job until certain conditions are met.
+
+        Blocks until the AnnotationImport.state changes either to
+        `AnnotationImportState.FINISHED` or `AnnotationImportState.FAILED`,
+        periodically refreshing object's state.
+
+        Args:
+            sleep_time_seconds (str): a time to block between subsequent API calls
+        """
+        while self.state.value == AnnotationImportState.RUNNING.value:
+            logger.info(f"Sleeping for {sleep_time_seconds} seconds...")
+            time.sleep(sleep_time_seconds)
+            self.__backoff_refresh()
+
+    @backoff.on_exception(
+        backoff.expo,
+        (labelbox.exceptions.ApiLimitError, labelbox.exceptions.TimeoutError,
+         labelbox.exceptions.NetworkError),
+        max_tries=10,
+        jitter=None)
+    def __backoff_refresh(self) -> None:
+        self.refresh()
+
+    @functools.lru_cache()
+    def _fetch_remote_ndjson(self, url: str) -> List[Dict[str, Any]]:
+        """
+        Fetches the remote ndjson file and caches the results.
+
+        Args:
+            url (str): Can be any url pointing to an ndjson file.
+        Returns:
+            ndjson as a list of dicts.
+        """
+        response = requests.get(url)
+        response.raise_for_status()
+        return ndjson.loads(response.text)
+
+    @classmethod
+    def _build_import_predictions_query(cls, file_args: str, vars: str):
+        query_str = """mutation testPredictionImportsPyApi($parent_id : ID!, $name: String!, $predictionType : PredictionType!, %s) {
```

Comment:
should this be called `testPredictionImportsPyPyi` still? Maybe `ModelErrorAnalysisPredictionImport`?